lib/UnoCore/Targets/Android/Android.uxl

```diff
@@ -238,6 +238,7 @@
     <Set Commands.Run="@(HOST_WIN32:Defined:Test('run.bat', 'bash run.sh'))" />
 
     <ProcessFile Name="app/build.gradle" />
+    <ProcessFile Name="app/proguard-rules.pro" />
```

Comment:
This file is compiler-generated and shouldn't be listed here.

Reply:
Is there a way to merge and or append to the compiler-generated one if a user-defined one is defined?

Reply:
Got an update coming 😄 ...